pbpy/pbdispatch.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
 import os
```

Comment:
Can you push a playable build on team city with this pbsync build to see if everything works?

Reply:
I am not sure how to test that, if you can, please do.

Reply:
Works, resolved.